app/views/dashboard/index.html.erb

```diff
@@ -53,6 +53,14 @@
       <p class="govuk-body">Prisoners whose case information has not yet been matched with nDelius.</p>
     </div>
   </div>
+  <div class="govuk-grid-column-one-third">
+    <div class="wrapper">
+      <h1 class="govuk-heading-s">
+        <%= link_to "See all handover cases", prison_summary_handovers_path(@prison.code), class:"govuk-link" %>
+      </h1>
+      <p class="govuk-body">All cases that are within the handover period or start the handover process to the community in the next 30 days.</p>
```

Comment:
I think this sentence is too long, and people will have trouble parsing it (I'm not even sure if the title/helper text is the right way to frame these).

```suggestion
      <p class="govuk-body">Cases that need handing over to the community soon.</p>
```

(I'm not a content designer, just suffering from hubris. It's possible our actual content designer looked at this, too. Hmm.)

Reply:
This content has been changed by our content designer – new commit on its way.